src/Model/Scope.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data\Model;
+
+use atk4\data\Exception;
+use atk4\data\Model;
+
+/**
+ * The root scope object used in the Model::$scope property
+ * All other conditions of the Model object are elements of the root scope
+ * Scope elements are joined only using AND junction.
+ */
+class Scope extends Scope\CompoundCondition
+{
+    /**
+     * @var Model
+     */
+    protected $model;
+
+    public function __construct(array $nestedConditions = [])
```

Comment:
should topmost/Model Scope be allowed to be instatiated from something else than Model?